instance/tests/models/test_database_server.py

```diff
@@ -223,6 +223,50 @@ def test_set_field_defaults(self):
         self.assertEqual(custom_mongodb_server.port, 1234)
 
 
+@override_settings(
+    DEFAULT_INSTANCE_MONGO_URL=None,
+    DEFAULT_MONGO_REPLICA_SET_NAME="test_name",
+    DEFAULT_MONGO_REPLICA_SET_USER="test",
+    DEFAULT_MONGO_REPLICA_SET_PASSWORD="test",
+    DEFAULT_MONGO_REPLICA_SET_PRIMARY="test.opencraft.hosting",
+    DEFAULT_MONGO_REPLICA_SET_HOSTS="test.opencraft.hosting,test1.opencraft.hosting,test2.opencraft.hosting"
+)
+class MongoDBReplicaSetManagerTest(TestCase):
+    """
+    Test cases for MongoDBReplicaSetManager.
+    """
+
+    def test__create_default_replica_set(self):
```

Comment:
@rocioar nit: double underscore: test__

Reply:
this is because _create_default is a private method, and starts with an underscore, the first underscore on the name of this tests it's just to divide the words, and the other one is part of the name of the function this is testing.